Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyCreateProduct.Codeunit.al

```diff
@@ -87,22 +87,7 @@ codeunit 30174 "Shpfy Create Product"
                             ProductPriceCalc.CalcPrice(Item, ItemVariant.Code, ItemUnitofMeasure.Code, TempShopifyVariant."Unit Cost", TempShopifyVariant.Price, TempShopifyVariant."Compare at Price");
                             TempShopifyVariant.Title := ItemVariant.Description;
                             TempShopifyVariant."Inventory Policy" := Shop."Default Inventory Policy";
-                            case Shop."SKU Mapping" of
```

Comment:
More could be refactored in this codeunit, but I'll only change things that I touch/need. And as I wanted to extract the create variant from item, I extracted this SKU mapping, as it's the same for every procedure.